scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -80,108 +79,112 @@
 y.value_counts(True)
```

Comment:
I didn't realize you need to individually add each comment in a review session (I was assuming they were all tracked/saved for some reason). Lesson learned! My additional comments are below.